swap/Cargo.toml

```diff
@@ -10,27 +10,26 @@ anyhow = "1"
 async-recursion = "0.3.1"
 async-trait = "0.1"
 atty = "0.2"
-backoff = { version = "0.2", features = ["tokio"] }
+backoff = { git = "https://github.com/ihrwein/backoff", rev = "9d03992a83dfdc596be26276d4e5c5254a4b11a2", features = ["tokio"] }
```

Comment:
There is no released version to depend on yet?
(Just asking to make sure we depend on the commit hash on purpose and it was not for testing.)

Reply:
@rishflab ?